config/routes.rb

```diff
@@ -302,10 +303,7 @@
   get '/admin/fix_results_selector' => 'admin#fix_results_selector', as: :admin_fix_results_ajax
   get '/admin/person_data' => 'admin#person_data'
   get '/admin/do_compute_auxiliary_data' => 'admin#do_compute_auxiliary_data'
```

Comment:
Why is this route not being deleted along with the others?

Reply:
Because this route is still used for import results page.

Reply:
Gotcha!